projects/plugins/boost/app/assets/src/js/pages/index/index.tsx

```diff
@@ -141,7 +141,12 @@ const Index = () => {
 								title={ __( 'Page Cache is unavailable', 'jetpack-boost' ) }
 								hideCloseButton={ true }
 							>
-								<p>{ __( 'Your website already has a page cache running on it powered by WordPress.com.', 'jetpack-boost' ) }</p>
+								<p>
```

Comment:
This came from merging with trunk 🤔 I'll investigate what happened here.

Reply:
This is strange, but what's happening is that we're fixing this. The commit hook is running prettier which is formatting this file. I guess whoever committed this change didn't run the prettier hook for some reason.